Dockerfile.txt

```diff
@@ -0,0 +1,44 @@
+# Dockerfile # Mapknitter
+# https://github.com/publiclab/mapknitter/
+# This image deploys Mapknitter!
+
+FROM ruby:2.4.6-stretch
+
+# Set correct environment variables.
+ENV HOME /root
+
+# Backported GDAL
+RUN echo "deb http://packages.laboratoriopublico.org/publiclab/ stretch main" > /etc/apt/sources.list.d/publiclab.list
+
+# Obtain key
+RUN apt-key adv --keyserver ipv4.pool.sks-keyservers.net --recv-keys BF26EE05EA6A68F0
+
+# Install dependencies
+RUN apt-get update -qq && apt-get install -y \
+  nodejs gdal-bin curl procps git imagemagick python-gdal zip
+
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && apt-get install -y npm
+RUN npm install -g bower
```

Comment:
This is deprecated, we use Yarn now

Reply:
Do we even need this file?

Reply:
The reason I renamed Dockerfile to Dockerfile.txt in this branch was that at "Cloud Build" trigger, it would pick the Dockerfile instead of the cloudbuild.yaml file where the deployment is defined.

Reply:
Ohh, I get it. We need to refactor this one, since we're no longer using Bower